src/main/java/com/researchspace/dao/customliquibaseupdates/StoreFileContentsHash_rsdev292.java

```diff
@@ -0,0 +1,60 @@
+package com.researchspace.dao.customliquibaseupdates;
+
+import com.researchspace.core.util.CryptoUtils;
+import com.researchspace.files.service.FileStore;
+import com.researchspace.model.FileProperty;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import liquibase.database.Database;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class StoreFileContentsHash_rsdev292 extends AbstractCustomLiquibaseUpdater {
+
+  private int fileCounter;
+  private FileStore fileStore;
+
+  @Override
+  protected void addBeans() {
+    fileStore = context.getBean("compositeFileStore", FileStore.class);
+  }
+
+  @Override
+  public String getConfirmationMessage() {
+    return String.format(
+        "Populated contentsHash column in the FileProperty table for %d entities.", fileCounter);
+  }
+
+  @Override
+  protected void doExecute(Database database) {
+    List<FileProperty> fileProperties = getAllFileProperties();
+    for (FileProperty fileProperty : fileProperties) {
+      if (fileProperty.getContentsHash() == null || fileProperty.getContentsHash().isEmpty()) {
```

Comment:
Could you use `StringsUtil.isBlank(...)`?

Reply:
Yeah good call, have updated it